tools/Status/Status.py

```diff
@@ -340,6 +349,15 @@ def render_status(show_paths, show_unidentified, **kwargs):
               default=None,
               help=("On a new screen, refresh jobs every 'watch' number "
                     "of seconds. Exit out with Ctl+C."))
+@click.option("--state-styles",
+              "input_state_styles",
```

Comment:
@nilsvu When trying this on wheeler, I had to specify `input_state_styles` in the config file rather than just `state_styles`. Do you know why that is?

It's not a big deal, I can just swap around the names, but was curious why chose the variable name and not the option name.

Reply:
Click uses the variable name as key in the `default_map` that it passes to the command. Just delete the `input_state_styles` name and pass it through as `state_styles` (see comment above).